packages/components/select/src/SelectTrigger.styles.tsx

```diff
@@ -17,11 +17,11 @@ export const styles = cva(
         success: 'ring-success',
       },
       disabled: {
-        true: 'disabled:bg-on-surface/dim-5 cursor-not-allowed text-on-surface/dim-3',
+        true: '!bg-on-surface/dim-5 cursor-not-allowed !text-on-surface/dim-3',
```

Comment:
Do we really need to use the `!` variant here?

Reply:
Yes, because of L7, where we define the `bg-surface` :disappointed:

Reply:
I tried this first, but well then compound logic had to be also done for `disabled` variant... Seemed simpler this way.But if it is blocking to you I can totally try again :+1:

Reply:
It's not a blocker on my side, but if we can avoid using the `!` 🤷